### PR TITLE
fix: show and hide raw content of detail

### PR DESCRIPTION
**Pull request by davidsoderberg**
Screenshot 2022-10-28 at 11 06 56
Screenshot 2022-10-28 at 11 07 03
Screenshot 2022-10-28 at 11 07 09
Screenshot 2022-10-28 at 11 07 16

### CODE REVIEW

File: apps/web/src/components/activity/ExecutionDetailShowRaw.tsx [hunk 1]
```diff
@@ -94,7 +93,7 @@ export const ExecutionDetailRawSnippet = ({ raw, onClose }) => {
         data-test-id="execution-detail-raw-snippet"
         language="json"
       >
-        {raw}
+        {JSON.stringify(JSON.parse(raw), null, 2)}
```
Comment (p-fernandez) [inline apps/web/src/components/activity/ExecutionDetailShowRaw.tsx:96]: I tried this but I wasn't able to work. Thanks for fixing it!

File: apps/web/src/components/activity/ExecutionDetailsModal.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function ExecutionDetailsModal({
       styles={{
         modal: {
           backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
-          width: '60%',
+          width: '90%',
```
Comment (p-fernandez) [inline apps/web/src/components/activity/ExecutionDetailsModal.tsx:37]: In the designs the width of the modal was set to a 60% of the screen size. I guess it is ok if we increase it.

File: apps/web/src/components/activity/ExecutionDetailsStepContent.tsx [hunk 3]
```diff
@@ -52,19 +53,21 @@ export const ExecutionDetailsStepContent = ({ step }) => {
...
                 onHideExecutionDetail={onHideExecutionDetail}
               />
             ))}
           </GridColContainer>
         </Grid.Col>
         <Grid.Col span={6}>
           <GridColContainer theme={theme}>
-            {!showTriggerSnippet && <ExecutionDetailTrigger step={step} />}
-            {showTriggerSnippet && executionDetailsRawSnippet && (
+            <When truthy={detailId.length === 0}>
```
Comment (p-fernandez) [inline apps/web/src/components/activity/ExecutionDetailsStepContent.tsx:65]: I find it more confusing to check that a string is empty by checking its length rather that strict checking empty string. Is there any edge case that this covers?

File: apps/web/src/components/activity/ExecutionDetailsStepContent.tsx [hunk 4]
```diff
@@ -52,19 +53,21 @@ export const ExecutionDetailsStepContent = ({ step }) => {
               <ExecutionDetail
                 key={`execution-detail-${executionDetail.id}`}
                 executionDetail={executionDetail}
-                showTriggerSnippet={showTriggerSnippet}
-                onShowExecutionDetail={onShowExecutionDetail}
+                showTriggerSnippet={detailId !== executionDetail.id}
+                onShowExecutionDetail={onShowExecutionDetail(executionDetail.id, executionDetail.raw)}
```
Comment (p-fernandez) [inline apps/web/src/components/activity/ExecutionDetailsStepContent.tsx:57]: My React might be a bit rusty but isn't this going to get executed? Shouldn't we pass a function declaration?
```suggestion
                onShowExecutionDetail={() => onShowExecutionDetail(executionDetail.id, executionDetail.raw)}
```

**Review by p-fernandez — APPROVED** [review]
Thanks for fixing this mess I did. 🙏🏻

File: apps/web/src/components/activity/ExecutionDetailsStepContent.tsx (hunk 4 above)
Comment (davidsoderberg) [reply apps/web/src/components/activity/ExecutionDetailsStepContent.tsx:57]: We do check how onShowExecutionDetail is implemented 😄

File: apps/web/src/components/activity/ExecutionDetailsStepContent.tsx (hunk 3 above)
Comment (davidsoderberg) [reply apps/web/src/components/activity/ExecutionDetailsStepContent.tsx:65]: I do not think so just the way I learned to check an empty string 😄

File: apps/web/src/components/activity/ExecutionDetailsModal.tsx (hunk 2 above)
Comment (davidsoderberg) [reply apps/web/src/components/activity/ExecutionDetailsModal.tsx:37]: Yeah this was needed to make the payload look better in the screenshots...

File: apps/web/src/components/activity/ExecutionDetailShowRaw.tsx (hunk 1 above)
Comment (davidsoderberg) [reply apps/web/src/components/activity/ExecutionDetailShowRaw.tsx:96]: No problem took me a while to understand how to resolve this 👍

File: apps/web/src/components/activity/ExecutionDetailsStepContent.tsx (hunk 3 above)
Comment (p-fernandez) [reply apps/web/src/components/activity/ExecutionDetailsStepContent.tsx:65]: Up to your choice then, though first thought I had was why `detailId` was an array. 😅

**Review by p-fernandez — APPROVED** [review]
🔥

**Pull request merged**